server/middlewares/validateAuth.js

```diff
@@ -0,0 +1,30 @@
+import { signinSchema } from '../schemas/authSchema';
+/**
+ * This class contains all methods
+ * required to validate
+ * authentication data.
+ */
+class validateAuth {
+  /**
+   * This method handle the signin request.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @param {object} next pass to next method.
+   * @returns {object} The error and some error messages .
+   */
+  static validateSignin(req, res, next) {
```

Comment:
Consider capitalizing the **i** as well to become **validateSignIn**.

Reply:
Done!